src/coordination/azure/Akka.Coordination.Azure/AzureLeaseOption.cs

```diff
@@ -51,20 +49,20 @@ public override void Apply(AkkaConfigurationBuilder builder, Setup? s = null)
                 sb.AppendLine($"lease-operation-timeout = {LeaseOperationTimeout.ToHocon()}");
             sb.AppendLine("}");
 
+            builder.AddHocon(sb.ToString(), HoconAddMode.Prepend);
+            
+            if(AzureCredential is null && ServiceEndpoint is null)
+                return;
```

Comment:
Code cleanup, shortcut `Apply` to not inject a `Setup` class if there is no need for it